client/src/views/page-components/ProductDetailPage/ProductDetail/ProductDetail.js

```diff
@@ -39,7 +20,9 @@ export default function ProductDetail({ $target, onSlideHandler, initialState })
   const deleteConfirmModal = new DeleteConfirmModal({
     $target,
     onDeleteHandler: () => {
-      console.log('삭제 API 연결');
+      ProductAPI.deleteProduct(this.state.id, this.state.user.accessToken).then(
+        (res) => (location.href = 'http://52.79.77.241:8080/')
+      );
     },
   });
```

Comment:
토큰 후에 redirect 하는 코드군요...!! 
res가 필요 없어 보입니다
자주 사용한다면 로직을 따로 빼도 되겠네요
```suggestion 
      const HOME_URL = 'http://52.79.77.241:8080/';
      const redirectToHome = () => window.location.href = HOME_URL

      ProductAPI.deleteProduct(this.state.id, this.state.user.accessToken).then(redirectToHome);
    },
  });
```